src/main/java/org/prebid/server/auction/StoredRequestProcessor.java

```diff
@@ -109,7 +110,8 @@ private Future<BidRequest> processAuctionStoredRequest(String accountId, BidRequ
 
         return storedRequestsToBidRequest(
                 storedDataFuture, bidRequest, bidRequestToStoredRequestId.get(bidRequest), impToStoredRequestId)
-                .map(this::generateBidRequestIdForApp);
+                .map(this::generateBidRequestIdForApp)
+                .map(resolvedRequest -> AuctionStoredResult.of(true, resolvedRequest));
```

Comment:
Issue says that we need to generate `tid`
- for `BidRequest` if `request.ext.prebid.storedrequest.id` exists
- for `imp` if `imp.ext.prebid.storedrequest.id` exists.

Implemented logic doesn't match requirements.

Reply:
https://github.com/prebid/prebid-server/issues/2381 was updated